plugins/org.jkiss.dbeaver.ext.tidb.ui/src/org/jkiss/dbeaver/ext/tidb/ui/internal/TiDBMessages.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2022 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jkiss.dbeaver.ext.tidb.ui.internal;
+
+import org.eclipse.osgi.util.NLS;
+
+public class TiDBMessages extends NLS {
+    private static final String BUNDLE_NAME = "org.jkiss.dbeaver.ext.tidb.ui.internal.TiDBMessages"; //$NON-NLS-1$
+
+    public static String tidb_connection_page_label_port;

```

Comment:
It looks like you do not need it anymore. This class and properties.

Reply:
Deleted.